src/eqProps.js

```diff
@@ -14,7 +14,7 @@ var equals = require('./equals');
  * @param {Object} obj1
  * @param {Object} obj2
  * @return {Boolean}
- *
+ * @deprecated since v0.18.0
```

Comment:
I'm pleased we agree! The problem with providing lots of sugar is that users have less exposure to the underlying general-purpose functions, so knowledge is less transferable. Sugar is best when those reading the code understand the "macro expansion" that will take place.